libminifi/include/core/yaml/YamlConfiguration.h

```diff
@@ -41,10 +42,12 @@ namespace core {
 class YamlConfiguration : public FlowConfiguration {
 
  public:
-  YamlConfiguration(std::shared_ptr<core::Repository> repo,
-                    std::shared_ptr<core::Repository> flow_file_repo,
+  YamlConfiguration(const std::shared_ptr<core::Repository> &repo,
```

Comment:
might have to do the same here to avoid removing const ( which you can do obviously )